argilla/docs/how_to_guides/annotate.md

```diff
@@ -0,0 +1,191 @@
+
+At the bottom of the annotation view, you can find the `Discard`, `Save as draft` and `Submit` buttons that will be enabled when at least one record is selected and, in the case of `Submit`, when all the required questions are annotated. To select specific records, you can click on the checkbox on the top left of each record. To select all records, you can click the checkbox in the bottom left of the preferences view. When discarded/saved as draft/submitted, **all the selected records** will change their status, so you can continue annotating the rest of the records.
+
+!!! Tip
+    Some specific situations in which you may want to use the bulk view are:
+
+    * Applying the same label to the results of a similarity search.
+    * Discarding all records in a specific language or that do not match a certain pattern or guidelines.
+    * Saving/submitting records with a suggestion score over a safe threshold.
+
+### Custom record view
+
+The bulk view also allows you to customize the record view, which can be particularly useful when your records are long.
+
+=== "Collapse records"
+
+    You can collapse the records to focus on the fields you are annotating. To do so, you can click on the `Record size` button at the bottom right of the preferences view. A dropdown will appear with the options `Expand records` and `Collapse records`.
+
+=== "Manage the number of records by page"
+
+    You can manage the number of records displayed by page. To do so, you can click on the `Page size` button at the bottom right of the preferences view. A dropdown will appear with the options `10`, `25`, `50` and `100`.
+
+## Understand patterns in your data
+
+!!! info "Main use cases"
+    * You have a large volume of data to annotate.
+    * You need to perform extensive cleaning.
+    * You have numerous and/or long fields.
+    * You are familiar with your data.
+
+The UI provides different features to help you understand patterns in your data and manage the annotation task more efficiently.
+
+### Search and highlight
+
+You can search for records with specific words in their field content. For this, you should click on the magnifying glass icon in the top right of the preferences view. A search bar will appear where you can type the word or words you are looking for. If your records have multiple fields, you will be able to select a specific field for the search or select `All` to search on all the fields. Pressing `Enter` will show the records that contain these words, highlighted in <span style="color:#ff675f">coral</span>, in the selected fields.
+
+### Order by similar record
```

Comment:
Not sure what a similar record means in this context. Perhaps it is good to empohasize text?
```suggestion
### Order by textual record similarity
```